students/jakielczyk_miroslaw/lesson_01_basics/Lesson_01.py.py

```diff
@@ -0,0 +1,104 @@
+# Sum of three numbers
+a = int(input("Enter number 1"))
+b = int(input("Enter number 2"))
+c = int(input("Enter number 3"))
+print ("Sum of these three numbers ",(a+b+c))
+
+# Area of right-angled triangle
+print ("Enter length of the base of a right-angled triangle")
+a=float(input())
```

Comment:
wrong spacing. There should be one single space around `=` sign for value assignment